wazo_dird/plugins/status/http.py

```diff
@@ -6,9 +6,23 @@
 
 
 class StatusResource(AuthResource):
-    def __init__(self, status_aggregator):
+    def __init__(self, status_aggregator, config):
         self.status_aggregator = status_aggregator
+        self._config = config
 
     @required_acl('dird.status.read')
     def get(self):
-        return self.status_aggregator.status(), 200
+        result = {
+            'bus_consumer': {
+                'status': self.status_aggregator.status()['bus_consumer']['status']
+            },
+            'rest_api': {
+                'status': self.status_aggregator.status()['rest_api']['status']
+            },
+            'master_tenant': {
+                'status': 'ok'
+                if self._config['auth'].get('master_tenant_uuid')
+                else 'fail'
+            },
+        }
+        return result, 200
```

Comment:
Why? Please read how status_aggregator work
You can look at wazo-chatd and/or almost all other services

Reply:
Actually I kept the original at the begining (like wazo-chatd and wazo-setupd); however the `test_config.py` was failing with `master_tenant` key not found. So I took a look at the code in `wazo-webhood`: [status-http](https://github.com/wazo-platform/wazo-webhookd/blob/master/wazo_webhookd/plugins/status/http.py#L15)